gcn/notices/fermi/gbm/alert.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "GCN Schema for Fermi GBM Alert Notices",
+  "description": "This schema is used to report Fermi GBM Alerts.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "../../core/AdditionalInfo.schema.json"
+    },
+    {
+      "$ref": "../../core/Alert.schema.json"
+    },
+    {
+      "$ref": "../../core/DateTime.schema.json"
+    },
+    {
+      "$ref": "../../core/Event.schema.json"
+    },
+    {
+      "$ref": "../../core/GeoLocBase.schema.json"
+    },
+    {
+      "$ref": "../../core/Reporter.schema.json"
+    },
+    {
+      "$ref": "../../core/Statistics.schema.json"
+    },
+    {
+      "$ref": "detectors.schema.json"
+    }
+  ],
+  "properties": {
+    "notice_type": {
+      "type": "string",
+      "description": "Notice title"
```

Comment:
Can this be an enum?  How many options are there for notice title?  Are you always going to refer back to GCN Classic notice types?  You don't need to be tied to those.

Reply:
Added `"const": "FERMI_GBM_ALERT"`.  This is redundant with `$schema` and the topic.  Thanks for pointing out.